hls4ml/backends/fpga/fpga_backend.py

```diff
@@ -414,7 +421,7 @@ def compute_conv1d_instructions(self, in_W, in_C, kernel_size=3, stride=1, pad=0
         for i in range(min_W):
             windows_int.append(int(''.join(str(p) for p in reversed(windows_bin[i])), 2))
 
-        return (min_W, windows_int)
+        return (min_W, windows_int, narrow)
```

Comment:
I don't understand why do you need to return the `narrow` state? Isn't it simpler to check `if min_w == in_W: use the narrow indexing function` and get rid of the `narrow` attribute later (the check determines the scale function to use). And could we call the narrow index function differently? You call it "narrow" but it returns unscaled index.

Reply:
So you are saying that if min_W == in_W you are in the "narrow" mode in all cases. I think that makes sense. Let me convince myself. 

As for the narrow name, I don't really have a strong opinion. Would "unscaled" be better?

(I'll also try to get the conflicts and precommits fixed.)

Reply:
I went ahead and made those changes. Let's see how the CI goes.